src/mlpack/tests/ann/not_adapted/gan_test.cpp

```diff
@@ -46,7 +46,7 @@ TEST_CASE("GANTest", "[GANNetworkTest]")
   double multiplier = 1;
 
   arma::mat trainData(1, 10000);
-  trainData.imbue( [&]() { return arma::as_scalar(RandNormal(4, 0.5));});
+  trainData.imbue( [&]() { return arma::as_scalar(RandNormal(4, 0.707));});
```

Comment:
```suggestion
  trainData.imbue( [&]() { return arma::as_scalar(RandNormal(4, std::sqrt(0.5)));});
```
That's more precise and I think compilers will optimize out the sqrt(constant) call anyway (example: https://stackoverflow.com/questions/21808271/why-do-square-roots-of-constants-known-at-compile-time-not-require-linking-the-m).